app/views/help.scala.html

```diff
@@ -100,146 +250,203 @@ <h3 class="govuk-heading-s">Step two: Transfer agreement</h3>
             </ul>
             <p class="govuk-body">
-                The transfer agreement is in two parts. The first part covers the requirements that records must be public, crown copyright and in english.
-            </p>
-            <img src="@routes.Assets.versioned("images/TA-1.png")" class="screenshots" alt="A list of checkboxes which need to be checked before continuing." />
-            <p class="govuk-body">
-                The second part of the transfer agreement is to confirm that the Departmental Records Officer (DRO) has signed off the sensitivity review, appraisal and selection decision.
-            </p>
-            <img src="@routes.Assets.versioned("images/TA-2.png")" class="screenshots" alt="A second list of checkboxes which need to be checked before continuing." />
-            <h3 class="govuk-heading-s">Step three: Upload your records</h3>
-            <p class="govuk-body">
-                To upload your records, you can either drag and drop a file or folder OR choose a file or folder for upload. Please ensure your internet connection is strong and stable. Before uploading please consider the following information:
-            </p>
-            <ul class="govuk-list govuk-list--bullet">
-                <li>You can upload one file or one top-level folder per transfer. However, your folder can contain multiple files and sub-folders</li>
-                <li>We cannot accept files that are password protected or encrypted</li>
-                <li>During private beta, we cannot accept zip files</li>
-                <li>Files such as thumbnail images (thumbs.db) and system files are not usually selected for permanent preservation. We strongly advise you do not upload these</li>
-                <li>During the upload process, TDR runs DROID to capture basic file-level metadata. Such as file path, file name, date (usually last modified), file format, and checksum</li>
-            </ul>
-            <img src="@routes.Assets.versioned("images/upload.png")" class="screenshots" alt="Upload page with a multi-select area for dropping your folder or selecting through a button." />
+                The transfer agreement is in two parts.
+            </p>
+            <ol class="govuk-body">
+                <li>The first part covers the requirements that records must be public, crown copyright and in English.
+                    Click each box to confirm and then 'Continue' when ready to proceed.</li>
+                <img src="@routes.Assets.versioned("images/standard-TA-1.png")" class="screenshots" alt="Transfer Agreement part 1 page" />
+                <li>The second part of the transfer agreement is to confirm that the Departmental Records Officer (DRO) has signed off the sensitivity review, appraisal and selection decision, and that the records are open for publication.
+                    (Closed records cannot currently be accepted whilst TDR is operating in private beta phase.)
```

Comment:
I think the full stop after "publication" might be a mistake as well as the full stop after "beta phase"; I think it's supposed to be after the closing bracket